src/workerd/api/node/buffer.c++

```diff
@@ -88,24 +88,8 @@ void SwapBytes(kj::ArrayPtr<kj::byte> bytes) {
   }
 }
 
-inline Encoding getEncoding(kj::StringPtr input) {
-  if (input == "utf8"_kj) {
-    return Encoding::UTF8;
-  } else if (input == "ascii"_kj) {
-    return Encoding::ASCII;
-  } else if (input == "latin1"_kj) {
-    return Encoding::LATIN1;
-  } else if (input == "utf16le"_kj) {
-    return Encoding::UTF16LE;
-  } else if (input == "base64"_kj) {
-    return Encoding::BASE64;
-  } else if (input == "base64url"_kj) {
-    return Encoding::BASE64URL;
-  } else if (input == "hex"_kj) {
-    return Encoding::HEX;
-  }
-
-  JSG_FAIL_REQUIRE(Error, kj::str("Invalid encoding: ", input));
+constexpr Encoding getEncoding(uint8_t input) {
+  return static_cast<Encoding>(input);
```

Comment:
Likely better just to skip this helper. Change the existing call sites to do the static_cast directly